llm/wit/golem-llm.wit

```diff
@@ -258,39 +260,59 @@ interface llm {
     delta(stream-delta),
     /// Converstation finished
     finish(response-metadata),
-    /// Conversation failed
-    error(error),
   }
 
   /// Represents an ongoing streaming LLM conversation
   resource chat-stream {
     /// Polls for the next chunk of stream events
-    get-next: func() -> option<list<stream-event>>;
+    poll-next: func() -> result<option<list<stream-event>>, error>;
```

Comment:
I think having the error in the stream was nicer, and more forward looking for being replaced with a WASI p3 `stream<stream-event>`. Why change it?

Reply:
as discussed in private: so that retries can happen with "default" usage and bindings when using wasm-rquickjs